docs/user-guide/Arktos-Release-Notes.md

```diff
@@ -0,0 +1,75 @@
+1. API Server/Core Server with Multi-Tenancy and Unified Pod Support
+1. Arktos VM Runtime Server
+
+### Features
+1. Multi-tenancy Features 
+     1. Introduce a new layer “tenant” before “namespace” in API resource URL schema, to provide a clear and isolated resource hierarchy for tenants.
+     1. Introduce a new API resource “tenant”, to keep tenant-level configurations and properties.
+     1. The metadata section of all exiting API resources has a new member: tenantName.
+     1. API Server, ClientGo, Scheduler, Controllers and CLI changes for the new resource model.
+1. Unified VM/Container Support:
+     1. Extend “pod” definition to both containers and VM. Now a pod can contain one VM, or one or more containers.
+     1. Enhance scheduler to schedule container pods and VM pods in the same way (unified scheduling).
+     1. Enhance kubelet to support multiple CRI runtimes (unified agent).
+     1. Implement a VM runtime server evolved from project Virtlet, with new features like VM reboot, snapshot, restore, etc.
+     1. Enhance kubelet to handle VM state changes and configuration changes.
+     1. Introduce a new API resource “action” and the corresponding handles (action framework) to support some VM specific actions which are not appropriate to be expressed as state machine changes, like reboot and snapshot.
+1. Artkos Integration with OpenStack Neutron
+     1. Arktos network controller integrate with neutron.
+     1. Arktos CNI plugins for neutron. 
+1. Arktos integration with Mizar
+     1. Arktos CNI plugins for mizar-mp.
+1. Scalability
+     1. Partitioned and scalable controller managers with active-active support.
+     1. API Server Partitioning (in progress)
+     1. ETCD partitioning (in progress)
+### Known Issues
+1. Go 1.13 is not supported yet.
+### Features in Planning
+1. Intelligent scheduling
+1. in-place resource update 
+1. QoS enforcement
+
+## Arktos Release v0.2 (3/30/2020)
+
+### Features Added
+1. Multi-tenancy Features
+     1. Tenancy short-path support 
+     2. Add Tenant Controller  
+     3. Tenancy-aware token Authenticator 
+     4. Tenancy-aware Cert Authenticator 
+     5. Tenancy-aware RBAC Authorizer   
+     6. Tenancy in kubeconfig context
+     7. Stabilization on multi-tenancy API Model  
+     8. More test and workaround fixes Added   
+
+2. Scalability Features
+     1. Multiple API Server Data Partitioning  
+     2. Endpoints consolidation changes for api server data parition 
+     3. Automatically set up test environments for data partitioned environment 
+     4. Workload Controller Manager running in parallel with Kube Controller Manager. Replicaset controller and deployment controller are running in Worload Controller Manager and can have multiple instances 
+     5. Added kube-up and start-kubemark for AWS 
+     
+3. Unifed VM/containers
+     1. Add support for primary runtime 
+     2. Add volume driver for OpenStack Cinder 
+     3. Fix issues on VM pod vCPU settings 
```

Comment:
on VM pod -> related to VM pod

Reply:
Thanks Deepak. I will address some of my changes in the next commit.
And for the order of the release note, this release-note.md I am adding is mainly created for the documentation website (table of content included which won't require users to scroll down on the page.)   We will be using the built in release function that comes with GitHub, so should address this concern there. Thanks!